server/backend/quiz/tests/test_views.py

```
@@ -0,0 +1,81 @@
+from rest_framework import status
+from users.models import User
+from quiz.models import Category, Question, Option, QuizAttempt, UserAnswer
+
+@pytest.mark.django_db
+class TestQuizViews:
+
+    client = APIClient()
+
+    def test_category_view_set(self):
+        category = Category.objects.create(name="Math", description="Math questions")
+        response = self.client.get(f'/quiz/')
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)